usermods/EleksTube_IPS/TFTs.h

```diff
@@ -141,14 +141,18 @@ class TFTs : public TFT_eSPI {
     h = read32(bmpFS); // height
     read16(bmpFS); // color planes (must be 1)
     bitDepth = read16(bmpFS);
-
+    bool monochromatic = false;
+    
     if (read32(bmpFS) != 0 || (bitDepth != 24 && bitDepth != 1 && bitDepth != 4 && bitDepth != 8)) {
       Serial.println(F("BMP format not recognized."));
       bmpFS.close();
       return false;
+    } else {
+      if (bitDepth == 1) monochromatic = true;
     }
 
     uint32_t palette[256];
+    uint32_t palette0 = 255;
     if (bitDepth <= 8) // 1,4,8 bit bitmap: read color palette
     {
```

Comment:
Coding style : use `(void) read32(bmpFS)` if you want to ignore return value of your function. This will avoid some compiler warnings especially for new MCUs. Also it makes the code clearer, as you explicitly say "yes I want to ignore the function return value this time".